### PR TITLE
Get Registry Notice PDF

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Call report soap service to get report.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetRegistryNoticeTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package ca.bc.gov.open.jag.efilingapi.filingpackage.filingpackageApiDelegateImpl;
+
+import ca.bc.gov.open.jag.efilingapi.Keys;
+import ca.bc.gov.open.jag.efilingapi.filingpackage.FilingpackageApiDelegateImpl;
+import ca.bc.gov.open.jag.efilingapi.filingpackage.service.FilingPackageService;
+import org.junit.jupiter.api.*;
```
Comment (akroon3r) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetRegistryNoticeTest.java:6]: do we need all of this? I'm assuming it's intellij

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/filingpackage/FilingpackageApiDelegateImpl.java [hunk 2]
```diff
@@ -83,6 +83,18 @@ public FilingpackageApiDelegateImpl(FilingPackageService filingPackageService) {
 
     }
 
+    @Override
+    public ResponseEntity<Resource> getRegistryNotice(BigDecimal packageIdentifier) {
+
+        logger.info("get payment receipt request received");
```
Comment (alexjoybc) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/filingpackage/FilingpackageApiDelegateImpl.java:89]:
```suggestion
        logger.info("get registry notice request received");
```

**Review by alexjoybc — CHANGES_REQUESTED** [review]
small thing on the log

**Review by alexjoybc — APPROVED** [review]
Everything is awesome!

yeah

**Pull request merged**